### ISSUE TITLE
Buoys fall through the sea

### ISSUE THREAD

**Issue by mole99** [rwengine/openrw#401]
As of now there are no buoys in the game, because they are falling through the water surface.

![bildschirmfoto von 2018-02-23 21-11-59](https://user-images.githubusercontent.com/3877395/36933040-f4a28d6e-1ed2-11e8-8cd1-348b7dd9d0c9.png)

If you start the Debug Mode and use the camera to got to the nearest waters, you can see them slowly falling down.

![bildschirmfoto vom 2018-03-03 11-05-47](https://user-images.githubusercontent.com/3877395/36933046-10c4b666-1ed3-11e8-991a-24917b9231aa.png)

As seen on the picture above, they should just float on the water and move with the waves.

**Comment by mole99** [comment]
Yes you are right, there is already some code to float in `tickPhysics` under `VehicleObject`. I will take a look at this.

**Comment by mole99** [comment]
This should be fixed by #440 
I hope it wasn't because of my bullet version, but we will see!

**Comment by JayFoxRox** [comment]
Just for the future: If you write "Closes #440" in your PR descriptions, then the issue will be closed automatically on merge (and a more meaningfull log entry is created automatically).

**Comment by mole99** [comment]
Thanks, I'll keep that in mind!
